includes/functions.php

```diff
@@ -154,3 +154,57 @@ function wc_gateway_ppec_get_transaction_fee( $order ) {
 
 	return $fee;
 }
+
+/**
+ * Saves the transaction details from the transaction response into a post meta.
+ *
+ * @since 2.0
+ *
+ * @param object $order    Order for which the transaction was made
+ * @param object $response Response from a transaction
+ * @param object $prefix   A prefix string which is empty for Reference Transactions and is 'PAYMENTINFO_0_' for Express Checkout
+ * @return void
+ */
+function wc_gateway_ppec_save_transaction_data( $order, $response, $prefix = '' ) {
```

Comment:
This is a very small detail, so feel free to disregard it. But what do you think about renaming `$response` here to something else like `$transaction` or `$order_details` for clarity?

This may be because I'm still relatively new to this codebase, but without context, I would have to look through this function code to get an idea of what is meant.